globus_automate_client/action_client.py

```diff
@@ -10,11 +11,34 @@
 _ActionClient = TypeVar("_ActionClient", bound="ActionClient")
 
 
+PRODUCTION_ACTIONS_BASE_URL = "https://actions.globus.org"
+
+_ENVIRONMENT_ACTIONS_BASE_URLS = {
+    None: PRODUCTION_ACTIONS_BASE_URL,
+    "prod": PRODUCTION_ACTIONS_BASE_URL,
+    "production": PRODUCTION_ACTIONS_BASE_URL,
```

Comment:
I guess we'll have to watch for both prod and production!  Without knowing which other services support which, it is safer to have both options.